pkg/config/msp.go

```diff
@@ -255,10 +508,24 @@ func pemEncodeX509Certificate(cert x509.Certificate) []byte {
 }
 
 func pemEncodeECDSAPrivateKey(priv *ecdsa.PrivateKey) ([]byte, error) {
-	privBytes, err := x509.MarshalPKCS8PrivateKey(priv)
+	privBytes, err := x509.MarshalECPrivateKey(priv)
 	if err != nil {
-		return nil, fmt.Errorf("marshalling PKCS8 private key: %v", err)
+		return nil, fmt.Errorf("marshalling ec private key: %v", err)
 	}
 
-	return pem.EncodeToMemory(&pem.Block{Type: "PRIVATE KEY", Bytes: privBytes}), nil
+	return pem.EncodeToMemory(&pem.Block{Type: "EC PRIVATE KEY", Bytes: privBytes}), nil
+}
```

Comment:
Can you explain this change? I'd like to understand the motivation.

PKCS#8 handles multiple algorithms - not just EC. Some clients (Java chaincode, for example) are able to deal with PKCS#8 keys but fail to decode SEC 1 keys.

Reply:
As mentioned in standup I was under the impression Fabric only supported EC private keys. After discussion in standup though I understand where you're coming from on this and will change it back to PKCS8